allensdk/brain_observatory/behavior/behavior_project_cache/behavior_ecephys_project_cache.py

```diff
@@ -0,0 +1,283 @@
+from functools import partial
+from typing import Optional, Union
+from pathlib import Path
+import logging
+
+from allensdk.brain_observatory.behavior.behavior_project_cache.project_apis.data_io import (  # noqa: E501
+    BehaviorEcephysProjectCloudApi)
+from allensdk.api.warehouse_cache.caching_utilities import call_caching
+
+
+class VisualBehaviorEcephysProjectCache(object):
```

Comment:
I think this should be named `VisualBehaviorNeuropixelsProjectCache`. The cache is associated with this specific data release, which is called "Visual Behavior Neuropixels". There was a degeneracy with VBO in that the project was called "Visual Behavior Ophys" and the data was behavior data plus ophys data. We don't have this degeneracy this time, so we have to make a choice. I think the only guarantee we can make to our future selves is that this cache matches the schema for the VBN data release. There is no guarantee that the next team to release behavior plus ecephys data will want *these* metadata tables and *this* specific NWB schema.

Reply:
renamed